### PR TITLE
Correctly select task in DAG Graph View when clicking on its name

### PR DESCRIPTION
**Pull request by jonhspyro**
Allowed taskName component to be clickable

### CODE REVIEW

File: airflow/www/static/js/dag/details/graph/DagNode.tsx [hunk 1]
```diff
@@ -126,10 +125,6 @@ const DagNode = ({
           label={taskName}
           isOpen={isOpen}
           isGroup={!!childCount}
-          onClick={(e) => {
-            e.stopPropagation();
```
Comment (bbovenzi) [inline airflow/www/static/js/dag/details/graph/DagNode.tsx:142]: We need this for opening and closing task groups. But we can wrap it in an if statement `if (!!onToggleCollapse) {...` and have both function inside. Therefore, the onclick will propagate to selecting the task if it is not a task group

Comment (jonhspyro) [reply airflow/www/static/js/dag/details/graph/DagNode.tsx:142]: So the code I deleted is making some other task not work?

When I tested, everything was working.
Can you give me some tips on how to solve this issue?

Comment (bbovenzi) [reply airflow/www/static/js/dag/details/graph/DagNode.tsx:142]: Yes, it was used to open and close task groups.

We just need to change onClick to:

```
          onClick={(e) => {
            if (!!childCount) {
              e.stopPropagation();
              onToggleCollapse();
            }
          }}

Comment (jonhspyro) [reply airflow/www/static/js/dag/details/graph/DagNode.tsx:142]: Can you check the changes I made? :) I think now it's working

**Review by jonhspyro — COMMENTED** [review]
Is this correct?

**Review by bbovenzi — APPROVED** [review]

**Review by jscheffl — APPROVED** [review]
Just a small formatting-glitch (indent is wrong, static check hit the wall) - can you correct? Then we could merge it. Rebase on fix would also be great.

**Comment by jscheffl** [comment]
Still there are some static checks. Can you fix them? Maybe you can also run them locally via `pre-commit`.

File: airflow/providers/fab/auth_manager/cli_commands/user_command.py
Comment (jscheffl) [inline airflow/providers/fab/auth_manager/cli_commands/user_command.py:1]: What is the reason to add a change of FAB to the UI change? Is this an accidental commit?
I'd propose to separate this to another PR to separate concerns if and to be individually cherry-picked and tracked with an issue ID.

Comment (bbovenzi) [reply airflow/providers/fab/auth_manager/cli_commands/user_command.py:1]: @jonhspyro mind moving this commit to another PR so we can merge this?

**Comment by pedro-cf** [comment]
Greetings since @jonhspyro  is MIA, is there any way to continue with this merge?

**Comment by jscheffl** [comment]
> Greetings since @jonhspyro is MIA, is there any way to continue with this merge?

I think if @pedro-cf is MIA... can you follow-up and clean the branch in another fork? Then we could merge. But seems some un-related changes are still there besides a conflict...

**Comment by pedro-cf** [comment]
> > Greetings since @jonhspyro is MIA, is there any way to continue with this merge?
> 
> I think if @pedro-cf is MIA... can you follow-up and clean the branch in another fork? Then we could merge. But seems some un-related changes are still there besides a conflict...

I've never contributed PRs to airflow sorry, was just hoping this could be merged somehow.

**Review by bbovenzi — APPROVED** [review]
I removed the user_command changes. lgtm
